package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "appwrite",
+  "homepage": "https://appwrite.io/support",
+  "description": "Appwrite backend as a service cuts up to 70% of the time and costs required for building a modern application. We abstract and simplify common development tasks behind a REST APIs, to help you develop your app in a fast and secure way. For full API documentation and tutorials go to [https://appwrite.io/docs](https://appwrite.io/docs)",
+  "version": "",
+  "license": "BSD-3-Clause",
+  "main": "index.ts",
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "devDependencies": {},
+  "dependencies": {
+    "typescript": "^3.6.4",
```

Comment:
TypeScript should be a `devDependency`, as mentioned above, it's a tool you use, but your consumers should be agnostic to this.